package/restream/package

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(restream)
+pkgdesc="A binary framebuffer capture tool for the reStream script"
+url=https://github.com/rien/reStream
+pkgver=0.0.0-c41b877
+timestamp=2021-01-01T14:55:28+01:00
+section=utils
+maintainer="Dan Shick <dan.shick@gmail.com>"
```

Comment:
https://github.com/toltec-dev/toltec/blob/testing/docs/package.md#maintainer You okay with being the one in charge of reviewing alongside one of us with merge power? Or would you prefer for one of us to own that responsibility for now?

Reply:
I'm happy to review updates for this package.